tools/interface/tests/ontology_interface_test.py

```diff
@@ -0,0 +1,64 @@
+import sys
+from os import path
+
+#add to python path to import local libraries
+sys.path.insert(1, '../../validators/ontology_validator/')
+sys.path.insert(1, '../../')
+
+from absl.testing import absltest
+
+from yamlformat.validator import external_file_lib
+from yamlformat import constants as validator_constants
+
+from interface import ontology_interface
+from interface.tests import test_resources 
+
+class OntologyTest(absltest.TestCase):
+
+    def setUp(self):
+        super(OntologyTest, self).setUp()
+        #self.basedir = RESOURCE_PATH
+        yaml_files = external_file_lib.RecursiveDirWalk(validator_constants.ONTOLOGY_ROOT)
+        self.ontology = ontology_interface.Ontology(yaml_files)
+
+    def testGetFieldsForTypeName(self):
```

Comment:
It is more readable, IMO if the tests are smaller and test one thing at a time.  If this test failed, it would be very difficult to determine what condition was unmet.